lib/async_hooks.js

```diff
@@ -223,7 +223,7 @@ const storageHook = createHook({
 class AsyncLocalStorage {
   constructor() {
     this.kResourceStore = Symbol('kResourceStore');
-    this.enabled = false;
+    this.enabled = true;
```

Comment:
If you change the default value of the flag, you need to add the instance to the array.
In this case, the code here https://github.com/nodejs/node/pull/31950/files#diff-0bb01a51b135a5f68d93540808bac801R249-R252 is not running and nothing works.

Reply:
I'd rather avoid doing this change. Lazy enabling done in `run*` methods postpones the performance penalty related with enabling the underlying async hook until the point when `AsyncLocalStorage` is really needed.

So, do we really need to change this?

Reply:
Oh, I actually meant to leave that as false--I made a bunch of changes that I rewrote a few times before pushing this commit and it looks like I missed this. :sweat_smile: